static/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -243,7 +243,7 @@ const SidebarDropdownActor = styled('button')`
       text-shadow: 0 0 6px rgba(255, 255, 255, 0.1);
     }
     ${UserNameOrEmail} {
-      color: ${p => p.theme.gray200};
+      color: ${p => p.theme.white};
```

Comment:
This OK in light mode?

Reply:
Yepp, the sidebar has a dark background in both light and dark mode.